src/tasks/get-groups.ts

```diff
@@ -31,7 +31,13 @@ export function getGroups({
     result.push(server);
   }
   if (allowedGroups.includes('client')) {
-    result.push(client);
+    const tasks = allowedGroups.includes('server')
```

Comment:
This seems brittle to me - what if we changed the task name in the future?

Reply:
In order to allow `hydrate` both `server` and `client` needs to be in the allowed groups

Reply:
Oh I see, this condition is inside a block that is checking for `client` and `server`

Can we add a comment to the code explaining that `hydrate` can only be run if both `client` and `server` is allowed?